purchases/build.gradle

```diff
@@ -69,6 +69,7 @@ dependencies {
     integrationTestImplementation 'androidx.appcompat:appcompat:1.4.1'
     integrationTestImplementation 'com.google.android.material:material:1.6.0'
     integrationTestImplementation 'androidx.constraintlayout:constraintlayout:2.1.3'
+    integrationTestImplementation("androidx.lifecycle:lifecycle-runtime-ktx:$lifecycleVersion")
```

Comment:
I was debating myself between `integrationTestImplementation ` and `androidTestIntegrationTestImplementation` so sounds good to me 👍 

BTW could you clarify what's the reasoning behind 👇? 
> we shouldn't be using the `integrationTest` flavor for anything public

Not sure if I fully understand as ultimately whatever dependency you add as `integrationTestImplementation ` or `androidTestIntegrationTestImplementation` will be only available from the integration tests, right? What am I missing? 🤔

Reply:
Well, basically I would like to keep the `integrationTest` flavor as small as possible. If there is any code needed, we should try to add it in the androidTest/androidTestIntegrationTest folder. This is in case we ever remove that flavor in the future.

> we shouldn't be using the integrationTest flavor for anything public

That was bad wording on my side 😅 . I mean't that nothing to be available in the `integrationTest` flavor if possible. We don't deploy the integrationTest flavor, so nothing there will eventually be public. My bad.

Reply:
> would like to keep the integrationTest flavor as small as possible. [...] This is in case we ever remove that flavor in the future.

Makes sense. Thanks for clarifying 👍 

> We don't deploy the integrationTest flavor, so nothing there will eventually be public.

I don't expect nothing under any test module / flavor to be public 😅